pkg/distributor/distributor.go

```diff
@@ -1270,9 +1270,9 @@ func (d *Distributor) limitsMiddleware(next PushFunc) PushFunc {
 	}
 }
 
-// nextOrCleanup returns a new PushFunc and a cleanup function that should be deferred by the caller.
+// NextOrCleanup returns a new PushFunc and a cleanup function that should be deferred by the caller.
```

Comment:
nit: Maybe note that this is used outside this codebase?

Reply:
👍 [b250cc4](https://github.com/grafana/mimir/pull/8080/commits/b250cc4230eeecff1b540fdcd80d51d37a47ea59).